### PR TITLE
Minor updates

### PR DESCRIPTION
**Pull request by xavierlepretre**
**Instructions: (For B9lab use only!)** Please fill out the following items in brackets `[ ]`, following the [b9lab-processes guideline](/docs/b9lab-processes.md).

Documentation content update for [Module X, Section Y]

This PR [... (elaborate here)]

[(Optional: Further notes for reviewers)]

### R1: Internal review (B9lab)

- Technical review
  - [ ] requested
  - [ ] completed
- Language review
  - [ ] requested
  - [ ] completed

### R2: External review

- Technical review
  - [ ] requested
  - [ ] completed
- Language review
  - [ ] requested
  - [ ] completed

### R3: Internal QA review (B9lab)

- Technical review
  - [ ] requested
  - [ ] completed
- Language review
  - [ ] requested
  - [ ] completed

### R4: Final external QA review

- Technical review
  - [ ] requested
  - [ ] completed
- Language review
  - [ ] requested
  - [ ] completed

### RC: Release Candidate

- [ ] Ready to be merged

### CODE REVIEW

File: hands-on-exercise/1-ignite-cli/1-ignitecli.md [hunk 1]
```diff
@@ -269,8 +272,39 @@ If you work with a machine using M1 architecture, the Docker image should allow
 3. Install Golang with `brew install go`.
 
 </PanelListItem>
+<PanelListItem number="2">
+
+If you are on Mac M1 with an aggressive anti-virus, it may prevent you from running Ignite, perhaps even within Docker. To circumvent this, you can recreate your Docker image for it to run within Docker **with the Intel CPU emulation**, like so:
```
Comment (happylolonly) [inline hands-on-exercise/1-ignite-cli/1-ignitecli.md:277]: added some comments in Discord

Comment (happylolonly) [reply hands-on-exercise/1-ignite-cli/1-ignitecli.md:277]: https://github.com/ignite/cli/issues/3741

Comment (xavierlepretre) [reply hands-on-exercise/1-ignite-cli/1-ignitecli.md:277]: Wait, did you not write that Docker without `--platform` also returned an `exit 1`?

Comment (coldice) [reply hands-on-exercise/1-ignite-cli/1-ignitecli.md:277]: @xavierlepretre is this resolved?

Comment (xavierlepretre) [reply hands-on-exercise/1-ignite-cli/1-ignitecli.md:277]: I think my version should still go because M1 in itself is not a blocker for others.

Comment (happylolonly) [reply hands-on-exercise/1-ignite-cli/1-ignitecli.md:275]: yes, without `--platform` it is not worked

Comment (happylolonly) [reply hands-on-exercise/1-ignite-cli/1-ignitecli.md:275]: image

**Pull request merged**